File: VB/WindowsFormsApplication79/CustomTileNavPaneViewInfo.vb
Imports DevExpress.XtraBars.Navigation
Imports System
Imports System.Collections.Generic
Imports System.Drawing
Imports System.Linq
Imports System.Reflection
Imports System.Text

Namespace WindowsFormsApplication79
    Public Class CustomTileNavPaneViewInfo
        Inherits TileNavPaneViewInfo

        Public Sub New(ByVal tileNavBar As TileNavPane)
            MyBase.New(tileNavBar)
            'arrowGlyph
        End Sub
        Protected Overrides Sub CreateArrowButton()
            Dim arrowGlyph As Bitmap = (TryCast(Me.TileNavPane, CustomTileNavPane)).ArrowGlyph
            If arrowGlyph IsNot Nothing Then
                GetType(TileNavPaneViewInfo).GetField("arrowGlyph", BindingFlags.NonPublic Or BindingFlags.Instance).SetValue(Me, arrowGlyph)
            End If
            MyBase.CreateArrowButton()
        End Sub
    End Class
End Namespace

File: CS/WindowsFormsApplication79/CustomTileNavPaneViewInfo.cs
using DevExpress.XtraBars.Navigation;
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Linq;
using System.Reflection;
using System.Text;

namespace WindowsFormsApplication79 {
    public class CustomTileNavPaneViewInfo : TileNavPaneViewInfo {
        public CustomTileNavPaneViewInfo(TileNavPane tileNavBar) : base(tileNavBar) {
            //arrowGlyph
        }
        protected override void CreateArrowButton() {
            Bitmap arrowGlyph = (this.TileNavPane as CustomTileNavPane).ArrowGlyph;
            if (arrowGlyph != null) {
                typeof(TileNavPaneViewInfo).GetField("arrowGlyph", BindingFlags.NonPublic | BindingFlags.Instance).SetValue(this, arrowGlyph);
            }
            base.CreateArrowButton();
        }
    }
}
